modules/openapi-generator/src/main/java/org/openapitools/codegen/AbstractGenerator.java

```diff
@@ -36,21 +40,73 @@
 
 public abstract class AbstractGenerator {
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractGenerator.class);
+    
+    /**
+     * Is the minimal-file-update option enabled?
+     * @return Option value
```

Comment:
[nits] Please insert blank comment line between the description and the list of tags.